drink/src/session.rs

```diff
@@ -154,7 +161,7 @@ impl<R: RuntimeWithContracts> Session<R> {
         Ok(Self {
             sandbox,
             mocks,
-            actor: R::default_actor(),
+            actor: Config::default_actor(),
```

Comment:
is this the only reason that `Session` is generic over `SandboxConfig` instead of `pallet_contracts::Config`?